src/main/java/org/truffleruby/core/module/ModuleNodes.java

```diff
@@ -1787,8 +1787,11 @@ protected RubyNode coerceToString(RubyNode name) {
         @Specialization
         protected RubyUnboundMethod instanceMethod(RubyModule module, String name,
                 @Cached BranchProfile errorProfile) {
+            final Frame callerFrame = getContext().getCallStack().getCallerFrameIgnoringSend(FrameAccess.READ_ONLY);
```

Comment:
Could we use ReadCallerFrameNode here, or is it involved?
If not, we can probably delay as anyway `instance_method` isn't really fast yet (and might not need to be).

Reply:
It seems this the last comment to fix (try to fix); will do this today\tomorrow

Reply:
Done: https://github.com/oracle/truffleruby/pull/2177/commits/01ef61ade721cc49daecf6a327160b26234e9e57